python/src/dynamixel_sdk/protocol1_packet_handler.py

```diff
@@ -536,7 +541,6 @@ def bulkReadTx(self, port, param, param_length):
                 i += 3
             port.setPacketTimeout(wait_length)
 
-        del txpacket[:]; del txpacket
```

Comment:
@roncapat 

is there any specific reason that for removing txpacket deletion

Reply:
It would be deleted by garbage collector after function return. Python has a reference count mechanism, so local function variables or objects will be deleted if not referenced outside